src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -0,0 +1,353 @@
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections;
+using System.ComponentModel;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+
+namespace System.Drawing
+{
+
+    public class ColorConverter : TypeConverter
+    {
+        private static string ValuesLock = "values";
+        private static System.ComponentModel.TypeConverter.StandardValuesCollection values;
+
+        public ColorConverter()
+        {
+        }
+
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            if (sourceType == typeof(string))
+            {
+                return true;
+            }
+            return base.CanConvertFrom(context, sourceType);
+        }
+
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            //if (destinationType == typeof(InstanceDescriptor))
+            //{
+            //    return true;
+            //}
```

Comment:
What's the purpose of this commented out code?  Is there a TODO here? Can it be deleted? Etc.